_posts/2015-06-23-hardly-anyone-knows-cd.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Hardly Anyone Knows Continuous Delivery
```

Comment:
Rephrase as "Not everyone knows what CD is". I'm just trying to mellow it down a bit :)